scripts/pre_commit_linter.py

```diff
@@ -1180,6 +1180,8 @@ def _lint_css_files(
             proc_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
         linter_stdout, linter_stderr = proc.communicate()
+        linter_stdout = linter_stdout.decode(encoding='utf-8')
```

Comment:
Try not to rename variables. Perhaps call the ones in the previous line encoded_linter_stdout, encoded_linter_stderr or similar.

Reply:
Done